lib/src/events/signal.dart

```diff
@@ -64,7 +64,20 @@ class Signal {
   }
 }
 
+enum EventSignalConfKey {
+  LongPressDuration,
+  LongPressShakingDistance,
+}
+
 class EventSignal<T> {
+  Map<EventSignalConfKey,Object> configure = {};
+  EventSignal();
+ factory   EventSignal.longPress(Duration duration,double distance) {
```

Comment:
Interesting approach to store the data inside the EventSignal... yet, Signals and MPS are event-bus driven.
They just `emit(data)`, there shouldn't be a persistency on external configuration... So i believe is better to stick the configuration for duration and distance into GDisplayObject and keep the code signature as it is.

Reply:
OK.